packages/eslint-plugin/src/rules/prefer-output-emitter-ref.ts

```diff
@@ -0,0 +1,31 @@
+import { Selectors } from '@angular-eslint/utils';
+import { createESLintRule } from '../utils/create-eslint-rule';
+
+type Options = [];
+
+export type MessageIds = 'preferOutputEmitterRef';
+export const RULE_NAME = 'prefer-output-emitter-ref';
+
+export default createESLintRule<Options, MessageIds>({
+  name: RULE_NAME,
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Use readonly `OutputEmitterRef` instead of `@Output()`',
```

Comment:
```suggestion
      description: 'Use `OutputEmitterRef` instead of `@Output()`',
```

…for clarity. This commit won’t be enough obviously, the docs will need to be regenerated I’m afraid